### DISCUSSION TITLE
2024/keyboards/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/shane0.github.io#60] [General]
# 2024/keyboards/

https://shanenull.com/2024/keyboards/

**Comment by shane0** [comment]
# boop

- the old blog https://shane0.gitlab.io/lektor/blog/
- imagine no time https://shanenull.com/stoicism/entropy/
- imagine a world without hypothetical thought

> humans are temporary pockets of low entropy
>> the universe - chapter 1 page 42


## atreus

- macos https://shop.keyboard.io/pages/atreus-getting-started

**Reply by shane0** [reply]
- desktop https://github.com/keyboardio/Chrysalis/tree/desktop

**Reply by shane0** [reply]
https://github.com/keyboardio/Chrysalis/releases/tag/v0.13.3

**Reply by shane0** [reply]
https://shanenull.com/2024/keyboards/

**Reply by shane0** [reply]
https://sites.google.com/view/vimkeyboard/home

**Reply by shane0** [reply]
https://shanenull.com/2023/keyboards/Chrysalis.pdf

**Comment by shane0** [comment]
# sid smith

- https://shanenull.com/stoicism/sid/

**Comment by shane0** [comment]
# retro

https://learn.nucamp.co/mod/assign/view.php?id=109

**Reply by shane0** [reply]
https://shanenull.com/2024/html/smile

**Reply by shane0** [reply]
Vintage html find the old js star menu and add Easter eggs

 https://shane0.neocities.org/
